plugin/server_audit/service_logger.h

```diff
@@ -0,0 +1,105 @@
+/* Copyright (C) 2012 Monty Program Ab
```

Comment:
In plugin/server_audit we have pretty much duplicated service_logger.h and file_logger.c?
Is that correct?

Reply:
They are not same, I did not want to change audit log code, so I made copy of original API and used new API on my implementation that uses C++. Audit plugin uses C and changing that is out of scope for this PR.